doc/topics/releases/releasecandidate.rst

```diff
@@ -2,53 +2,103 @@
+The builds should include the latest version of the operating system that is currently available.
 
 .. note::
 
-    Follow the instructions on https://repo.saltproject.io/,
-    but insert ``salt_rc/`` into the URL between the hostname
-    and the remainder of the path.
+   Beginning with the 3005 (Phosphorus) release of Salt, the Salt Project is
+   changing its packaging system to Tiamat. Any new operating systems added in 3005
+   will only have Tiamat packages. The Salt Project will phase out the old Salt
+   package builds for currently supported operating systems by 3007. See
+   `What is Tiamat? <https://docs.saltproject.io/salt/install-guide/en/latest/topics/upgrade-to-tiamat.html#what-is-tiamat>`_
+   for more information.
 
-    For Redhat Python 3
+To install release candidate packages:
+
+Follow the instructions for your operating system on https://repo.saltproject.io/,
+but insert ``salt_rc/`` into the URL between the hostname and the remainder
+of the file path.
 
-    .. code-block:: bash
 
-        baseurl=https://repo.saltproject.io/salt_rc/py3/redhat/$releasever/$basearch/
+Tiamat packages
+---------------
+For RedHat:
 
-    For Ubuntu Python 3 (replace os_version, with ubuntu version. For example 20.04)
+.. code-block:: bash
 
-    .. code-block:: none
+    baseurl=https://repo.saltproject.io/salt_rc/salt/py3/redhat/$releasever/$basearch/
 
-        deb https://repo.saltproject.io/salt_rc/py3/ubuntu/<os_version>/amd64 focal main
+For Ubuntu, replace the ``<os_version>`` variable with number of the Ubuntu
+version and ``<codename>`` for the release codename.
 
-    For Debian Python 3 (replace os_version, with debian version. For example 10)
+.. code-block:: none
 
-    .. code-block:: none
+    deb https://repo.saltproject.io/salt_rc/salt/py3/ubuntu/<os_version>/amd64 <codename> main
 
-        deb https://repo.saltproject.io/salt_rc/py3/debian/<os_version>/amd64 buster main
+For example, for the 22.04 release of Ubuntu, codename Jammy Jellyfish:
 
-The OSs that will be built for each RC release are the latest version of each OS on https://repo.saltproject.io
+.. code-block:: none
 
+    deb https://repo.saltproject.io/salt_rc/salt/py3/ubuntu/22.04/amd64 jammy main
 
-.. FreeBSD
 
-Installing Using Bootstrap
-==========================
+For Debian, the syntax is identical to Ubuntu. For example, for the version 10
+(Buster) release:
+
+.. code-block:: none
+
+    deb https://repo.saltproject.io/salt_rc/salt/py3/debian/10/amd64 buster main
+
+
+
+Classic packages
+----------------
+For RedHat:
+
+.. code-block:: bash
+
+    baseurl=https://repo.saltproject.io/salt_rc/py3/redhat/$releasever/$basearch/
+
+For Ubuntu, replace the ``<os_version>`` variable with number of the Ubuntu
+version and ``<codename>`` for the release codename.
+
+.. code-block:: none
 
+    deb https://repo.saltproject.io/salt_rc/py3/ubuntu/<os_version>/amd64 <codename> main
+
+For example, for the 22.04 release of Ubuntu, codename Jammy Jellyfish:
```

Comment:
22.04 will not be supported on the classic packages. I think we should use Ubuntu 20 instead for the example.

Reply:
Good catch. I've fixed this now.